### PR TITLE
fix(ScoreSummary): Add labels to informative icons

### PR DESCRIPTION
**Pull request by mstykmshy**
### Overview

- Unhides checkmark and x icons in ScoreSummary with the `colorfulIcons` prop and gives them an aria label

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [LEGO-3244]
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change
- [ ] This PR includes testing instructions tests for the code change
- [ ] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions

Instructions in monolith PR:

1. Complete a cj exam (for example `/exams/journeys/full-stack-engineer/paths/fscj-22-interview-prep/parts/1`) and move to the results screen
2. Navigate the results using VoiceOver and ensure labels are announced for the checkmark or x icons next to each unit

The icons in the list of units tested should be announced, since they communicate information (whether you got 70% of the questions in that unit right). The icons in the section above next to the "X correct/ X to work on" section shouldn't be announced since they don't add any information that isn't communicated by the text.

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/35408)  | [Monolith Env](https://pr-35408-monolith.dev-eks.codecademy.com/) |

[LEGO-3244]: https://codecademy.atlassian.net/browse/LEGO-3244?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.306-alpha.a980b3.0<br/>@codecademy/gamut-labs@35.11.4-alpha.a980b3.0<br/>@codecademy/styleguide@64.2.3-alpha.a980b3.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6438194bb0c91e0534419d21--gamut-preview.netlify.app>https://6438194bb0c91e0534419d21--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6438194bb0c91e0534419d21>Deploy Logs</a></p>

**Review by dreamwasp — APPROVED** [review]

File: packages/gamut-labs/src/ScoreSummary/index.tsx [hunk 1]
```diff
@@ -80,12 +80,16 @@ const renderSubScores = ({
...
+                  aria-hidden={false}
+                  aria-label="Checkmark"
                 />
               ) : (
                 <DeleteFilledIcon
                   minWidth={16}
                   mr={12}
                   color="feedback-error"
+                  aria-hidden={false}
+                  aria-label="X"
```
Comment (katiezutter) [inline packages/gamut-labs/src/ScoreSummary/index.tsx:92]: I think we might want to have the aria labels as something like 'passed' and 'failed' to make them more descriptive to what the meaning is

Comment (mstykmshy) [reply packages/gamut-labs/src/ScoreSummary/index.tsx]: that makes sense, updated now!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.306-alpha.ff1ef7.0<br/>@codecademy/gamut-labs@35.11.4-alpha.ff1ef7.0<br/>@codecademy/styleguide@64.2.3-alpha.ff1ef7.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6442c6395131152c578bab01--gamut-preview.netlify.app>https://6442c6395131152c578bab01--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6442c6395131152c578bab01>Deploy Logs</a></p>

**Review by katiezutter — APPROVED** [review]

**Pull request merged**